garage/misc/wrappers.py

```diff
@@ -0,0 +1,88 @@
+"""Gym env wrappers"""
+import cv2
+import gym
+import numpy as np
+
+from collections import deque
+from gym.spaces import Box
+
+
+class RepeatActionWrapper(gym.core.Wrapper):
+    """Gym env wrapper for repeating action for n frames"""
+
+    def __init__(self, env, frame_to_repeat):
+        super(RepeatActionWrapper, self).__init__(env)
+        self.frame_to_repeat = frame_to_repeat
+
+    def step(self, action):
+        for i in range(self.frame_to_repeat):
+            obs, reward, done, info = self.env.step(action)
+        return obs, reward, done, info
+
+    def reset(self):
```

Comment:
Isn't this covered by the superclass?

Reply:
When reset is not added, it complains 
`WARN: <class 'garage.envs.wrappers.repeat_action.RepeatAction'> doesn't implement 'reset' method, which is required for wrappers derived directly from Wrapper. Deprecated default implementation is used.`